### DISCUSSION TITLE
Volume control on LG Ultrafine 360 32"(32UN880)

### DISCUSSION THREAD

**Discussion by gr8radah** [MonitorControl/MonitorControl#731] [Q&A]
Works great on my LG monitor but there is a tiny flaw in changing volume.
It stops at level 4 and 29 of OSD scale when I try to change volume continuously so I need to press volume up/down twice at 4 and 29.
This is not a big deal but it would be perfect if it is fixed in the next release.

**Comment by waydabber** [comment]
hey @gr8radah - are you using the standard media keys or custom shortcuts? Thank you!

**Reply by waydabber** [reply]
I cannot seem to reproduce this issue and I am not sure what is happening exactly.

Is this happening on the v4.0.0-rc1? Can you give a little more info on how to reproduce this? Thanks!

**Reply by gr8radah** [reply]
I am using the standard media keys of Apple magic keyboard and Logitech MX keys.
I have to press volume keys twice to pass volume level 4 of OSD scale to 3 or 5.
The version is v4.0.0-rc1 which seems to be the latest. Thanks vm

**Reply by waydabber** [reply]
Hmm. Does OSD scale 3 to 5 refer to the macOS chiclets or the display's own OSD? Is the `Use fine OSD scale for volume` option enabled? Thanks!

<img width="842" alt="Screen Shot 2021-10-22 at 7 21 55" src="https://user-images.githubusercontent.com/37590873/138397595-195012c2-db0a-485a-b00f-e44cddee9e87.png">

**Reply by gr8radah** [reply]
I meant the display's OSD scale. I've enabled the option. Thanks.
